tests/integration/test_images_list.sh

```diff
@@ -28,6 +34,7 @@ setup () {
 
 teardown () {
     set +e
+    ${DOCKER} rm testImageOut
```

Comment:
One more minor nit: should use a proper container name (formatted xx-xx-xx), perhaps add it as a variable to the list at the top.